src/pages/ProductDetail/ArticleRight/ArticleRight.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import DetailTable from './DetailTable';
+import './ArticleRight.scss';
+
+function ArticleRight(props) {
+  const [count, setCount] = useState(0);
+
+  const increaseCount = () => setCount(count + 1);
```

Comment:
```suggestion
  const increaseCount = () => setCount(prevCount => prevCount + 1);
```
setState  함수안에 인자로 콜백함수를 넣어줄 수 있습니다.
그 콜백함수는 인자로 이전의 state값을 전달받고 return 한 값이 새로운 state가 됩니다!
이전 state 값을 활용해서 새로운 state를 만들때는 위와 같은 방식도 참고해서 적용해보세요!

Reply:
넵 감사합니다!!